d3a_api_client/redis_aggregator.py

```diff
@@ -84,6 +84,8 @@ def _events_callback_dict(self, message):
         elif "event" in payload and payload["event"] == "finish":
             self._on_finish(payload)
 
+        self.on_event_or_response(payload)
+
```

Comment:
in the case of `redis_aggregator` , all events were already passing through this `events_callback_dict, so I added the call to the new method after the ones already implemented.